rero_ils/modules/libraries/jsonschemas/libraries/library-v0.0.1.json

```diff
@@ -439,6 +439,12 @@
             },
             "address": {
               "$ref": "#/definitions/address_block"
+            },
+            "extra": {
```

Comment:
if it is only used for the `notifications`, I think it is better to have some thing like `notification_additional_information` instead of `extra`? 
or simply `additional_information`

Reply:
At this time, all data from shipping address and billing address sections are only use into notifications.
But could be use somewhere else if needed. I use "extra" as field_id because it's smaller as variable name (and we are limited to 80 chars).